data/base/shaders/button.frag

```diff
@@ -8,18 +10,36 @@ uniform vec4 teamcolour;
 uniform int tcmask;
 uniform bool alphaTest;
 
+#if __VERSION__ >= 130
```

Comment:
I'm not sure if using __VERSION__ directly is the best idea, as GLES GLSL uses a different number space for __VERSION__ than desktop GL. Eg GLES 3.1 has __VERSION__ == 310. Maybe instead use some define called 'LEGACY' or 'CORE' or even 'MODERN'?

Reply:
So this actually works just fine for GLES, and is functionally equivalent to the (more explicit):
`#if (!defined(GL_ES) && __VERSION__ >= 130) || (defined(GL_ES) && __VERSION__ >= 300)`
Because GLSL ES versions are `100` (which we want to take the appropriate branch here) or `300`+ (which we want to take the >= 130 branch here).

IMHO, using the variables that the GLSL standards provide is better than new custom defines. Anyone with shader experience can just look at the shaders to review them, without having to *also* dig into the Warzone C++ source to find where and when new defines are set.

I do think, however, there's a lot of merit to being more explicit (even when _technically_ unnecessary).

Reply:
It works for distinguishing between legacy/core and GLES2/GLES3, but once you start using __VERSION__ for more than that, it breaks down. But, I guess I'm thinking too far ahead already. We can improve this later. Feel free to commit.

Reply:
Ahh. Yeah, we can certainly expand this later if we need to segment off shader behavior based on other settings / properties. For simple things like this (how to specify in/out parameters), which *can* properly utilize __VERSION__, seems reasonable to use it.